website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
I moved the incentives code up. I can look into adding a second save if we want.